tools/explorer/pkg/stellar/stellar.go

```diff
@@ -73,10 +74,18 @@ func New(seed, network string, signers []string) (*Wallet, error) {
 		log.Warn().Msg("to enable escrow account recovery, provide atleast 3 signers")
 	}
 
+	// mimimumBalance can be calculated as following
+	// Minimum Balance = (2 + # of entries) * base reserve
+	// entries is the amount of operations are required to setup the account
+	// we have 3 trustline operations for assets, and an equally number of operations for
+	// the lenght of the signers
+	minimumBalance := float64((2 + 3 + len(signers))) * 0.5
```

Comment:
the amount of trustlines is currently 2, as we only support 2 assets right now, but will be 3 in the near future, so this comment is wrong at the moment. Regardless, the amount of supported assets is known here since we have the map. So we can just replace the static "3" here with `len(w.assets)` (after moving to a function). This means we won't have to touch this when a new asset is added, and allows us to have a different amount of assets when we are running on testnet or mainnet

Reply:
Actually, a rounding error cannot occur here. Since signers is always an even number?